.github/workflows/testing.yml

```diff
@@ -55,35 +55,39 @@ jobs:
           curl -L https://tarantool.io/release/2/installer.sh | sudo bash
           sudo apt install -y tt
 
+      - name: Setup tt environment
+        run: tt init
+
       - name: Setup Tarantool ${{ matrix.tarantool }}
-        if: matrix.tarantool != '2.x-latest'
+        if: matrix.tarantool != 'master'
         uses: tarantool/setup-tarantool@v2
         with:
           tarantool-version: ${{ matrix.tarantool }}
 
-      - name: Get Tarantool 2.x latest commit
-        if: matrix.tarantool == '2.x-latest'
+      - name: Get Tarantool master commit
+        if: matrix.tarantool == 'master'
         run: |
           commit_hash=$(git ls-remote https://github.com/tarantool/tarantool.git --branch master | head -c 8)
           echo "LATEST_COMMIT=${commit_hash}" >> $GITHUB_ENV
         shell: bash
 
-      - name: Cache Tarantool 2.x latest
-        if: matrix.tarantool == '2.x-latest'
+      - name: Cache Tarantool master
+        if: matrix.tarantool == 'master'
         id: cache-latest
         uses: actions/cache@v3
         with:
-          path: "${GITHUB_WORKSPACE}/bin"
+          path: |
+            ${{ github.workspace }}/bin
+            ${{ github.workspace }}/include
           key: cache-latest-${{ env.LATEST_COMMIT }}
 
-      - name: Setup Tarantool 2.x latest
-        if: matrix.tarantool == '2.x-latest' && steps.cache-latest.outputs.cache-hit != 'true'
+      - name: Setup Tarantool master
+        if: matrix.tarantool == 'master' && steps.cache-latest.outputs.cache-hit != 'true'
         run: |
-          tt init
           sudo tt install tarantool master
 
-      - name: Add Tarantool 2.x latest to PATH
-        if: matrix.tarantool == '2.x-latest'
+      - name: Add Tarantool master to PATH
+        if: matrix.tarantool == 'master'
         run: echo "${GITHUB_WORKSPACE}/bin" >> $GITHUB_PATH
```

Comment:
Don't we need to change this `GITHUB_WORKSPACE` too?

Reply:
No, we don't. The environment variable could be used in run script context.